.github/workflows/combine-fossa-report.yml

```diff
@@ -0,0 +1,126 @@
+name: Combine FOSSA Reports
+
+on:
+  workflow_dispatch:
+
+jobs:
+  wait-for-fossa-report-generation:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        repo: [
+          {name: "DaticalDB-installer", ref: "master",owner: "Datical"},
+#         {name: "ephemeral-database", ref: "DAT-18147",owner: "liquibase"}, #TODO: producing html report instead of csv. CSV report already uploaded in s3 to unblock the combine-fossa-reports job
+          {name: "drivers", ref: "master",owner: "Datical"},
+#         {name: "protoclub", ref: "DAT-18147",owner: "Datical"}, #TODO: producing html report instead of csv. CSV report already uploaded in s3 to unblock the combine-fossa-reports job
```

Comment:
@sayaliM0412 do we need to remove the TODOs?

Reply:
No, they need to be addressed. They are still producing `HTML` responses instead of `CSV`. When we click on the link from the run https://github.com/liquibase/ephemeral-database/actions/runs/10325875398/job/28588340050#step:11:4 you can see the correct `csv` report. But when a `cat` is done on the same file it produces `html` report:
https://github.com/liquibase/ephemeral-database/actions/runs/10325875398/job/28588340050#step:11:23

I have currently uploaded a `csv` format report for both in the s3 bucket.